### PR TITLE
feature:#107:  Thumbnail Update and Lead Terminology for the assignees

### LINKED ISSUE #107: Thumbnail Update and Lead Terminology for the assignees
This issue addresses two key improvements in our project management tool: updating the thumbnail representation of assignees and revising the terminology used for the project lead. The current thumbnail display lacks consistency and clarity, hindering quick identification of assignees. Additionally, the term 'key' used to represent the project lead is ambiguous and lacks descriptive value. This update aims to enhance clarity and consistency throughout the tool by improving the visual representation of assignees' thumbnails and replacing 'key' with 'displayName' for the project lead.

### PR DESCRIPTION
**Pull request by riginoommen**
### Description

Updated the default project lead name from 'key' to 'displayName' for clarity and consistency. Additionally, this commit introduces provisions for displaying the image of the assignee, enhancing the user interface and providing visual context alongside the assignee's display name.

The change aims to provide clearer and more intuitive naming conventions throughout the codebase, improving maintainability and understanding for developers.

### Screenshots

1. Thumbnail for the assignee and used the displayNames if available with the Jira
Screenshot

### Related Issue

Fixes #107

### Checklist before requesting a review

- [x] Self-reviewed own code
- [x] Verified that the code builds perfectly fine on my local system
- [x] Added tests proving the fix is effective or feature works
- [x] Commented code, particularly in hard-to-understand areas
- [x] Verified that code follows the style already available in the repository
- [x] Made corresponding changes to the documentation

### CODE REVIEW

File: plugins/jira-dashboard/src/components/JiraTable/columns.tsx [hunk 1]
```diff
@@ -75,18 +75,30 @@ export const columns: TableColumn[] = [
...
+    field: 'fields.assignee.displayName',
     highlight: true,
     type: 'string',
     width: '10%',
 
     render: (issue: Partial<Issue>) => {
-      if (issue.fields?.assignee?.name) {
+      if (issue.fields?.assignee?.displayName) {
+        return (
+          <div style={{ display: 'flex', alignItems: 'center' }}>
```
Comment (GLundh) [inline plugins/jira-dashboard/src/components/JiraTable/columns.tsx:86]: Can we use `Avatar` from `backstage/core-components` for consistency? I prefer the rounder look that is used elsewhere :)

Comment (riginoommen) [reply plugins/jira-dashboard/src/components/JiraTable/columns.tsx:86]: Sure

Comment (riginoommen) [reply plugins/jira-dashboard/src/components/JiraTable/columns.tsx]: Updated it

**Comment by riginoommen** [comment]
Requesting the review

**Comment by fridajac** [comment]
Hi @riginoommen! Thank you for your contribution! I've looked at your PR and added some comments. 

Unfortunately for us (Axis), we can not access the assignee image from the frontend, since it requires authentication towards Jira in the backend. However, let's go with your change and my team can fix this later (similar to how we do it with the project avatar).

**Comment by riginoommen** [comment]
> Hi @riginoommen! Thank you for your contribution! I've looked at your PR and added some comments. 
> 
> Unfortunately for us (Axis), we can not access the assignee image from the frontend, since it requires authentication towards Jira in the backend. However, let's go with your change and my team can fix this later (similar to how we do it with the project avatar). 

Thanks for the review @. I am looking to see the comments in PR. However i can't find any comments in PR

**Comment by fridajac** [comment]
> > Hi @riginoommen! Thank you for your contribution! I've looked at your PR and added some comments.
> > Unfortunately for us (Axis), we can not access the assignee image from the frontend, since it requires authentication towards Jira in the backend. However, let's go with your change and my team can fix this later (similar to how we do it with the project avatar).
> 
> Thanks for the review @fridajac . I am looking to see the comments in PR. However i can't find any comments in PR

Ah sorry, they were pending! Now they should be visible.

File: plugins/jira-dashboard/src/components/JiraTable/columns.tsx [hunk 2]
```diff
@@ -75,18 +75,31 @@ export const columns: TableColumn[] = [
   },
   {
     title: 'Assignee',
-    field: 'fields.assignee.name',
+    field: 'fields.assignee.displayName',
     highlight: true,
     type: 'string',
     width: '10%',
```
[the remark this answers is not in the document]
Comment (riginoommen) [reply plugins/jira-dashboard/src/components/JiraTable/columns.tsx]: updated the fields width

File: plugins/jira-dashboard/src/components/JiraTable/columns.tsx [hunk 3]
```diff
@@ -75,18 +75,31 @@ export const columns: TableColumn[] = [
...
 
     render: (issue: Partial<Issue>) => {
-      if (issue.fields?.assignee?.name) {
+      if (issue.fields?.assignee?.displayName) {
+        return (
+          <div style={{ display: 'flex' }}>
+            <Avatar
```
[the remark this answers is not in the document]
Comment (riginoommen) [reply plugins/jira-dashboard/src/components/JiraTable/columns.tsx:88]: updated

File: plugins/jira-dashboard/src/components/JiraTable/columns.tsx [hunk 4]
```diff
@@ -75,18 +75,31 @@ export const columns: TableColumn[] = [
...
+              picture={issue.fields?.assignee?.avatarUrls['48x48']}
```
[the remark this answers is not in the document]
Comment (riginoommen) [reply plugins/jira-dashboard/src/components/JiraTable/columns.tsx]: done

**Comment by riginoommen** [comment]
Fixed the comments requesting to please review the code

**Review by fridajac — APPROVED** [review]
Great job! Thank you a lot for contributing :+1:

**Pull request merged**